.travis.yml

```diff
@@ -21,26 +21,30 @@ matrix:
       compiler: gcc
 
     - os: linux
-      env: TEST_SUITE=testtravis
+      env: TEST_SUITE=testinstall GEN_COVERAGE=yes
```

Comment:
It would be simpler to set `GEN_COVERAGE=yes` onc in the global env section, and then explicitly set `GEN_COVERAGE=no` in the one job that needs it.

Or perhaps change it to `NO_COVERAGE`, and don't set it except for that one job. This way, we can't forget to enable it for e.g. AppVeyor (and perhaps other CI services, such as Jenkins, which might use `etc/ci.sh`). I.e. I propos that we turn coverage on by default, and one has to turn it off explicitly; instead of the reverse this PR does right now.